yt/data_objects/selection_data_containers.py

```diff
@@ -871,6 +871,13 @@ def __init__(self, data_source, conditionals, ds=None,
                 raise RuntimeError(
                     "Cannot use both base_object and data_source")
             data_source=base_object
+        if isinstance(data_source, YTCutRegion):
+            self.__init__(
+                data_source.base_object,
+                data_source.conditionals + conditionals,
```

Comment:
I think `ensure_list(conditionals)` needs to happen before this

Reply:
Totally right!